modules/UI/util/MessageHandler.js

```diff
@@ -373,6 +373,15 @@ var messageHandler = {
 
     enablePopups: function (enable) {
         popupEnabled = enable;
+    },
+
+    /**
+     * Returns true if dialog is opened
+     * false otherwise
+     * @returns {boolean} isOpened
+     */
+    isDialogOpened: function () {
```

Comment:
Very minor thing. I would rename this to isDialogVisible, which fits better the way we name such methods in general.